readme.md

```diff
@@ -25,7 +25,7 @@ Your Answer:
 Describe at a high level how we use jQuery to submit a form via AJAX.
```

Comment:
Noticed you left Question 1 blank. How are you feeling about AJAX? Included an answer for the question below. Happy to sit down and talk about it. This is a key JS concept that you'll want to be familiar with.

```js
$.ajax({
  url: "http://kittengifs.com/gifs"
  method: "get"
})
  .then(function(results){
    console.log(results.length)
  });
```